modules/installation-launching-installer.adoc

```diff
@@ -159,6 +159,8 @@ You can run the `create cluster` command of the installation program only once,
 
 ifndef::osp,rhv,vsphere[* Configure an account with the cloud platform that hosts your cluster.]
 
+ifdef::rhv[* Open the port `54322` in the installer machine.]
```

Comment:
The text should read:

> Open the port `54322` from the machine running the installer to the oVirt Engine. 

This gets rid of the ambiguity of where this port needs to be open.